FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Tea Jeremic on 19.8.21..
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal final class FeedImageMapper {
+	private struct RemoteObject: Decodable {
+		var items: [RemoteFeedImage]
+		var feedImages: [FeedImage] {
+			return items.map { $0.feedImage }
+		}
+	}
+
+	private struct RemoteFeedImage: Decodable {
+		let image_id: UUID
+		let image_desc: String?
+		let image_loc: String?
+		let image_url: URL
+
+		var feedImage: FeedImage {
+			return FeedImage(id: image_id, description: image_desc, location: image_loc, url: image_url)
+		}
+	}
+
+	static private let ValidStatusCode = 200
```

Comment:
Suggestion:
There're different practices how developers name constants. In swift it is more common to treat constants as properties and name it in camel case starting with a small letter. Some people prefer using capitalized snake case (THIS_IS_A_CONSTANT). 
At the same time naming a constant with camel case starting with a **capital** letter can confuse other developers as usually it is types that are name this way. To avoid this confusion it is better to use either **small** first letter or some other styling.

```suggestion
	static private let validStatusCode = 200
```